mr/src/main/java/org/apache/iceberg/mr/mapred/IcebergInputFormat.java

```diff
@@ -47,7 +51,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class IcebergInputFormat<T> implements InputFormat<Void, T> {
+public class IcebergInputFormat<T> implements InputFormat<Void, T>, CombineHiveInputFormat.AvoidSplitCombination {
```

Comment:
Can you put the same comments on here as in Hiveberg to explain why we've had to do these things?